packages/terra-date-picker/src/DatePicker.jsx

```diff
@@ -394,104 +400,57 @@ class DatePicker extends React.Component {
       selectedDateInPicker = this.state.selectedDate;
     }
 
-    const portalPicker = (
-      <ReactDatePicker
-        {...customProps}
-        selected={selectedDateInPicker}
-        value={formattedValue}
-        onBlur={this.handleBlur}
-        onChange={this.handleChange}
-        onChangeRaw={this.handleChangeRaw}
-        onClickOutside={this.handleOnClickOutside}
-        onFocus={this.handleOnInputFocus}
-        onSelect={this.handleOnSelect}
-        required={required}
-        customInput={(
-          <DateInput
-            onCalendarButtonClick={this.handleOnCalendarButtonClick}
-            inputAttributes={inputAttributes}
-            isIncomplete={isIncomplete}
-            isInvalid={isInvalid}
-            required={required}
```

Comment:
I noticed that the portalPicker has `required` set here by PR [#923](https://github.com/cerner/terra-framework/pull/923/files#diff-06f07d3cca7012107347dc63be00c147R415) whereas popupPicker below doesn't have it set. After talking to @BenBoersma it was not added for popupPicker most likely by mistake. Consolidating portalPicker and popupPicker should sync them up.

Reply:
nice catch! update: https://github.com/cerner/terra-framework/pull/1042/commits/0d9fa2162c5263155d82285505180d5af8892939